assets/js/lcp-beacon.js

```diff
@@ -332,14 +332,14 @@ class RocketLcpBeacon {
 		if (document.readyState !== 'loading') {
 			setTimeout(() => {
 				instance.init();
-			}, 500);
+			}, window.rocket_lcp_data.delay);
```

Comment:
Do we clear our cache with update?
If No, this would be a problem because this value will be undefined and throw console error because the js file is updated but the page itself is not updated and doesn't include this delay property.

I'd recommend guarding against this, what do u think?

```suggestion
			}, window.rocket_lcp_data.delay ?? 500);
```